File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys

t = int(sys.stdin.readline())
for i in range(t):
    a, b = map(int, sys.stdin.readline().rstrip().split())
    if a > b:
        max = a
        min = b
    else:
        max = b
        min = a
    while min > 0:
        max, min = min, max % min
    gcd = max
    lcm = a * b // gcd
    sys.stdout.write(str(lcm) + '\n')

File: 백준/Bronze/1934. 최소공배수/최소공배수.cc
#include <iostream>

using namespace std;

int main() {
    int t, a, b;
    cin >> t;
    for (int i = 0; i < t; i++) {
        cin >> a >> b;
        int min, max;
        if (a < b) {
            min = a;
            max = b;
        } else {
            min = b;
            max = a;
        }
        int plus = max;
        for (int j = 0; j < min; j++) {
            if (max % min == 0) {
                cout << max << '\n';
                break;
            } else {
                max += plus;
            }
        }
    }
}